bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiContentConverter.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bsblan.internal.api;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * Utility class to create JSON content.
+ *
+ * @author Peter Schraffl - Initial contribution
+ */
+public class BsbLanApiContentConverter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BsbLanApiContentConverter.class);
+
+    public static String toJson(BsbLanApiContent request) {
+        Gson gson = new Gson();
+        return gson.toJson(request);
+    }
+
+    @Nullable
+    public static <T> T fromJson(String content, Class<T> resultType) {
+        try {
+            Gson gson = new Gson();

```

Comment:
Why are you creating a new instance each time? Just make a static final instance that you can reuse.

Reply:
Changed it, thanks for the hint. Think I did it to ensure thread safety (without noticing that Gson is already thread safe :) )